### DISCUSSION TITLE
Best way to get rid of black tiles

### DISCUSSION THREAD

**Discussion by RISCfuture** [kubilus1/autoortho#223] [Q&A]
Just installed and tried AutoOrtho in XP12 (on Windows 12); it works great except I get lots of black terrain tiles on the ground that take a long time to "fill in" with the proper texture (or don't fill in at all). Are there settings I can tweak to minimize these black tiles? (a better provider, change cache size, etc.)

AMD 5950X, 64 GB RAM, RTX 4090, 1GBps up/down fiber

**Comment by dpdurst** [comment]
I don't think you meant Windows 12.. lol 
I've not seen black tiles but have seen dark green tiles filled in on spots a tile is missing. Cache size is only what is places on your PC so it doesn't have to keep fetching it. Default provider is Bing from what I understand, However there is nothing keeping you from trying other providers from the pull down list. I've seen some issues in Alaska off and on, and a few in Arizona but normally the next time I start up and fly those issues are a one off issue. I think your seeing what the rest of us does, and its the nature of the best so to say..
